DeepMRSeg/deepmrseg_test.py

```diff
@@ -1,7 +1,7 @@
 
 ################################################ DECLARATIONS ################################################
 __author__	 = 'Jimit Doshi'
-__EXEC_NAME__	 = "deepmrseg_test"
+__EXEC_NAME__	 = "deepmrseg_test_v3D"
```

Comment:
Lets keep the exec name to be deepmrseg_test

Reply:
Fixed